src/Model/CustomField.php

```diff
@@ -14,19 +14,9 @@ class CustomField
     function processData($data)
     {
 
-        $data['Type'] = "";
-        if (isset($data['Date']))
-            $data['Type'] = "Date";
-        if (isset($data['Number']))
-            $data['Type'] = "Number";
-        if (isset($data['Decimal']))
-            $data['Type'] = "Decimal";
-        if (isset($data['Boolean']))
-            $data['Type'] = "Boolean";
-        if (isset($data['Text']))
-            $data['Type'] = "Text";
+        $data['Value'] = $data[$data['Type']];
```

Comment:
Does this work? You've deleted the code that sets data['Type'] as far as I can see.

Reply:
Oops thanks for catching that, have added it back in.